src/java.base/share/conf/security/java.security

```diff
@@ -153,6 +153,60 @@ RestrictedSecurity.NSS.140-2.securerandom.provider = SunPKCS11-NSS-FIPS
 RestrictedSecurity.NSS.140-2.securerandom.algorithm = PKCS11
 #endif
 
+#if defined linux-x86 || defined linux-ppc || defined aix-ppc || defined windows
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.name = OpenJCEPlusFIPS FIPS 140-3 Cryptographic Module
```

Comment:
Is there significance to the naming difference as compared with NSS.140-2?
Why not `OpenJCEPlusFIPS Cryptographic Module FIPS 140-3` for consistency?

Reply:
Changed to suggestion.